Makefile

```diff
@@ -35,46 +35,39 @@ ifneq ($(TAGS),)
 TAGS_OPT 		= -tags ${TAGS}
 endif
 
-
 ut-local:
-	GO111MODULE=off go test -gcflags=-l -v `go list ./pkg/... | grep -v pkg/mtls/crypto/tls | grep -v pkg/networkextention`
+	GO111MODULE=on go test -gcflags=-l -v `go list ./pkg/... | grep -v pkg/mtls/crypto/tls | grep -v pkg/networkextention`
```

Comment:
does go test will use the modules in vendor?

Reply:
yes